systemvm/debian/etc/cron.daily/clear_cache

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# clear memory cache to ultimately reduce swapping
+
+sync && echo 1 > /proc/sys/vm/drop_caches
```

Comment:
I understand what you are doing there, but it is the Linux kernel which will remove files from it's case if it needs this memory for something else.

I wouldn't mess with this.

Reply:
As recommended amount of memory for a Debian host is 4GB as we are working **so** far below that, I would say that we are operating well outside 'normal' practices.  Personally, I have long questioned the use of Debian, and would have thought FreeBSD was a better choice.  wrt to memory, the minimum memory for Debian is 96MB.

at 256MB RAM we can't really afford almost ANY caching of disk to memory.  If we started at 4GB then i wouldn't dream of touching this.

The specific, highly replicable issue that is designed to fix, is that the cached disk access is not given back, instead the host starts swapping memory to disk, quickly making the VR unusably due to the CPU usage that this causes (see  https://github.com/apache/cloudstack/issues/3039)

I hope you can see why this sub-optimal fix is proposed to remedy the massively sub-optimal host spec that the VR is trying to operate in.

Reply:
This is safe in general, AND we are should not be concerned with disk latency, since we are doing network IO 99% of time.